node/mock-answer.ts

```diff
@@ -1,5 +1,5 @@
-import path = require('path');
-import fs = require('fs');
+import * as path from 'path';
```

Comment:
`import ... require` has slightly different semantics than `import ... from`.  My understanding is that `require` executes the module as the CommonJS and AMD `require` function does so that any side effects from the module are applied. `import ... from` is the normal way to import a module in TypeScript.